glfw/src/Window.zig

```diff
@@ -162,8 +163,10 @@ pub const Hint = enum(c_int) {
     /// Context client API minor version hint.
     context_version_minor = c.GLFW_CONTEXT_VERSION_MINOR,
 
-    /// Context client API revision number hint.
-    context_revision = c.GLFW_CONTEXT_REVISION,
+    // TODO: The docs don't specify GLFW_CONTEXT_REVISION as a hint, only as an attribute; is there a reason this is here? Commented to make current
+    // draft work, which is mostly copying from the docs
```

Comment:
The source backs you up here. GLFW_CONTEXT_REVISION is not a hint

Reply:
@silversquirl Is that the same case for `context_no_error = c.GLFW_CONTEXT_NO_ERROR`?